client/app/bundles/course/users/components/tables/ManageUsersTable.tsx

```diff
@@ -71,220 +81,358 @@ const translations = defineMessages({
   },
   changeRoleSuccess: {
     id: 'course.users.ManageUsersTable.changeRoleSuccess',
-    defaultMessage: "Successfully changed {name}'s role to {role}.",
+    defaultMessage: "Updated {name}'s role to {role}.",
   },
   changeRoleFailure: {
     id: 'course.users.ManageUsersTable.changeRoleFailure',
-    defaultMessage: "Failed to change {name}'s role to {role}.",
+    defaultMessage: "Failed to update {name}'s role to {role}.",
   },
   changeTimelineSuccess: {
     id: 'course.users.ManageUsersTable.changeTimelineSuccess',
-    defaultMessage:
-      "Successfully changed {name}'s timeline algorithm to {timeline}.",
+    defaultMessage: "Updated {name}'s reference timeline to {timeline}.",
   },
   changeTimelineFailure: {
-    id: 'course.users.ManageUsersTable.changeTimelineFailure',
+    id: 'course.users.ManageUsersTable.changeTimelineSuccess',
```

Comment:
Typo key?

Reply:
My bad, this was a really dumb thing. Fixed this and also for `bulkChangeTimelineFailure`.